src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -154,6 +154,10 @@
     <value>Close tabs after index {0}</value>
     <comment>{0} will be replaced with a number</comment>
   </data>
+  <data name="CopySuffix" xml:space="preserve">
+    <value> (Copy)</value>

```

Comment:
This is the teeniest nit, but could you keep the space and parentheses outside of the localized text? (like, inject the "copy" string into that context programmatically). Mildly worried that the localization team may forget to add the space or the parentheses or something like that.

Reply:
Oh, thanks for letting me know! Fixed